BasicRdl/Views/Dialogs/BooleanParameterTypeDialog.xaml

```diff
@@ -4,7 +4,7 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:dx="http://schemas.devexpress.com/winfx/2008/xaml/core"
              xmlns:dxlc="http://schemas.devexpress.com/winfx/2008/xaml/layoutcontrol"
-             xmlns:items="clr-namespace:CDP4CommonView.Items;assembly=CDP4CommonView"
+             xmlns:items="clr-namespace:CDP4CommonView.Items;assembly=CDP4Composition"
```

Comment:
why do these changes appear as part of this PR? they were taken care of in a previous commit?

Reply:
It was rebased and these commits just show up here.